storage/client_test.go

```diff
@@ -156,11 +156,15 @@ func (m *multiTestContext) Start(t *testing.T, numStores int) {
 }
 
 func (m *multiTestContext) Stop() {
-	m.clientStopper.Stop()
-	for _, s := range m.stoppers {
+	stoppers := append([]*util.Stopper{m.clientStopper}, m.stoppers...)
+	stoppers = append(stoppers, m.transportStopper)
```

Comment:
does `transportStopper` have to be at the end, or it be inlined above? if it has to be at the end, can you add a comment explaining why?